plugins/kubernetes-backend/src/service/router.ts

```diff
@@ -62,15 +62,16 @@ export const makeRouter = (
   router.use(express.json());
 
   // TODO error handling
-  router.get('/services/:serviceId', async (req, res) => {
+  router.post('/services/:serviceId', async (req, res) => {
     const serviceId = req.params.serviceId;
-
+    const authTokens: AuthTokens = req.body.auth;
```

Comment:
We likely need to pass on the entire request object to whatever part of the backend is responsible for authenticating requests. The mapping would probably be something like `(req, service, cluster) -> credentials` or something of the sort. Not sure exactly where that would happen though.

Still happy to merge this PR and see how we can untangle things afterwards